_data/site.yaml

```diff
@@ -1,11 +1,11 @@
-title: cloud.gov
-email: support@cloud.gov
+title: Agency Name
```

Comment:
Let's keep the cloud.gov info in.

Reply:
This is only showing up because we reset the branch to 11ty.  The PR to migrate data should clean this up just fine.

Reply:
See: https://github.com/cloud-gov/site/pull/17/commits/a1960c5f5afde919a313ada884059f77cc6df291#diff-5baa7a616344814920a8388f6ed5cd65bc704743d1900f35d2f244f0cfa1fdcd